kie-wb/kie-wb-webapp/pom.xml

```diff
@@ -1818,6 +1816,49 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.4</version>
+        <executions>
+          <execution>
+            <id>copy-wildfly-config</id>
+            <phase>process-resources</phase>
+            <configuration>
+              <tasks>
+                <copy
+                  file="${project.build.directory}/wildfly-${as.version}/standalone/configuration/standalone-full.xml"
+                  tofile="${project.build.directory}/wildfly-${as.version}/standalone/configuration/standalone.xml"
+                  overwrite="true"/>
+              </tasks>
+            </configuration>
+            <goals>
+              <goal>run</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.4</version>
```

Comment:
Same as above, should be possible to delete the version tag and use the one coming from `kie-parent-metadata`.